editoast/editoast_authz/src/builtin_role.rs

```diff
@@ -12,6 +12,16 @@ pub enum BuiltinRole {
     InfraRead,
     #[strum(serialize = "infra:write")]
     InfraWrite,
+
+    #[strum(serialize = "document:read")]
+    DocumentRead,
+    #[strum(serialize = "document:write")]
+    DocumentWrite,
+
+    #[strum(serialize = "electrical_profile:read")]
+    ElectricalProfileRead,
+    #[strum(serialize = "electrical_profile:write")]
+    ElectricalProfileWrite,
```

Comment:
electrical profiles don't need their roles since it doesn't make sense for a user to manipulate them if they're not allowed to manage infras. You can just use `InfraRead/Write`